pybamm/__init__.py

```diff
@@ -107,6 +107,8 @@ def version(formatted=False):
     SolverError,
     ShapeError,
     ModelWarning,
+    UndefinedOperation,
```

Comment:
just semantics, but I think `UndefinedOperationError` would be more consistent with the other errors